src/Vizkit3dWorld.hpp

```diff
@@ -19,31 +19,38 @@
 namespace vizkit3d_world {
 
 typedef std::map<std::string, vizkit3d::RobotVisualization*> RobotVizMap;
+static QApplication* app = NULL;
```

Comment:
Either make it class-scope static, or extern. Having it as namespace-scope static makes no sense.

Moreover, Qt already keeps a global with *the* QApplication instance. Don't remember how you can get it, but you can

Reply:
That would be `QApplication::instance()`.. You should not be keeping your own pointer to the instance

Reply:
👍